src/api/animals/Animals1Dash.js

```diff
@@ -84,7 +84,7 @@ const AnimalDash = () => {
 
         <Grid item xs={12} sm={6} md={4} lg={3}>
           <CardRouting
-            cardName="Cat Facts"
+            cardName="Animal Cat Facts"
             navigateTo="AnimalCatFacts"

```

Comment:
Animals

Reply:
Sorry my Bad I have made the necessary changes